### ISSUE TITLE
Feld für Faxnummer/E-Mail

### ISSUE THREAD

**Issue by littleyoda** [adiehl/hdgdldsgvo#1]
Bei der Firmenanschrift wäre ein weiteres Feld hilfreich, wo man E-Mail oder Fax eintragen kann.
z.B. "per Fax: xxxxxx" oder "per E-Mail: xxx@xxxx.eample"

**Comment by adiehl** [comment]
Stimmt, guter Punkt, nehme ich gerne so auf. Dann kann man es später auch an eine Fax-API anbinden.
Wird im nächsten Release berücksichtigt.

**Comment by adiehl** [comment]
Habe ich nun so umgesetzt - soll die Faxnummer dann in die Adresszeile mit eingefügt werden?

**Comment by littleyoda** [comment]
Ich fände es toll, wenn es im Adressfeld irgendwo in der Art "Per FAX: 9999-9999" auftauchen würde.
